src/aap_eda/wsapi/routes.py

```diff
@@ -8,8 +10,10 @@
     [path("ansible-rulebook", consumers.AnsibleRulebookConsumer.as_asgi())]
 )
 
-router = URLRouter(
+wsapi_router = URLRouter(
     [
         path(f"{settings.API_PREFIX}/ws/", wsapi_router),
     ]
 )
+
+router = AllowedHostsOriginValidator(DrfAuthMiddlewareStack(wsapi_router))
```

Comment:
@bzwei What would the recommended value for ALLOWED_HOSTS when the Activation container connects into the eda-server? Would that change between podman and Kubernetes?

Reply:
We will need to look into it for prod env. For my test in podman it works without any change. The default value is '*' for dev env.